docs/src/components/PageHeader.js

```diff
@@ -61,7 +63,12 @@ export default function ComponentHeader({
       {description && <Markdown text={description} />}
       {defaultCode && (
         <Box marginTop={8}>
-          <MainSection.Card cardSize="lg" showCode={false} defaultCode={defaultCode} />
+          <MainSection.Card
+            cardSize="lg"
+            showCode={false}
+            defaultCode={defaultCode}
+            shaded={shadedCodeExample}
```

Comment:
what is this doing?            `shaded={shadedCodeExample}`

Reply:
allows the page header example to have a shaded background

Reply:
Gotcha, this 
![image](https://user-images.githubusercontent.com/10593890/111230494-16ef6400-85be-11eb-97ae-8ef779f0f901.png)